public/scripts/power-user.js

```diff
@@ -1562,6 +1563,12 @@ function loadPowerUserSettings(settings, data) {
         power_user.tokenizer = tokenizers.GPT2;
     }
 
+    // Clean up old/legacy settings
+    if (power_user.import_card_tags !== undefined) {
```

Comment:
I felt we should respect the choice of existing users.
For new users - let them see the popup.

Reply:
._____.

Existing users might have just been annoyed by the badly designed popup which was especially spammy for multi character import and clicked "Never" out of desparation. But would love to actually have a reasonable default setting while still not seeing the popup. If they knew about it.  
Before the choice was "Always blocking popup" or "no import".  
I can see several reasons why people who chose "no import" didn't actually want that option.

But well. We can still implement a mapping and just hide the new options for those people. I just think they'll never see it, which is a pity.